### ISSUE TITLE
Does not work with tfoot

### ISSUE THREAD

**Issue by AidasK** [simonsmith/jquery.stickyTableHeader#7]
Error:
![image](https://user-images.githubusercontent.com/2088484/57772155-6a23d180-771d-11e9-888c-89cdbb61c356.png)
Uncaught TypeError: Cannot read property '0' of undefined

this line: 
https://github.com/simonsmith/jquery.stickyTableHeader/blob/master/src/index.js#L97

should be changed to:
this.$table.children('thead');

**Comment by simonsmith** [comment]
Thanks for the report

Do you have an example of the markup and plugin initialisation code for reproduction?

**Comment by AidasK** [comment]
Is there a fiddle example with imported `jquery.stickyTableHeader` to make it quicker?
Layout is:
```html
<table>
<thead>
...
</head>
<tbody>
...
</tbody>
<tfoot>
... summary columns, not sticky ...
</tfoot>
</table>
```

**Comment by simonsmith** [comment]
If you could fork this - https://codesandbox.io/s/9488w8jz04

Thanks

**Comment by AidasK** [comment]
success: https://codesandbox.io/embed/l7m8399n99

**Comment by AidasK** [comment]
tfoot contains th elements

**Comment by simonsmith** [comment]
@AidasK Thanks! I'll take a look at this later

**Comment by simonsmith** [comment]
Published in `1.1.1` - https://www.npmjs.com/package/jquery-sticky-table-header/v/1.1.1
